linter files

tingamapuro04 opened the pull request:
Add files for microverse linters

tingamapuro04 committed:
linter files

Add files for microverse linters

tingamapuro04 committed:
Add linter configuration

tingamapuro04 committed:
Add skeleton

tingamapuro04 committed:
Add about page

tingamapuro04 committed:
Add css to the homepage

tingamapuro04 committed:
about page

tingamapuro04 committed:
Add css for About page

tingamapuro04 committed:
Add css for the Home Page

tingamapuro04 committed:
Add styling for the home page

tingamapuro04 committed:
Add about

tingamapuro04 committed:
add styling for about page desktop

tingamapuro04 committed:
Linter corrections

tingamapuro04 committed:
Linter corrections

tingamapuro04 committed:
Linter corrections

tingamapuro04 committed:
Merge pull request #2 from tingamapuro04/About-page

About page

tingamapuro04 commented:
In this project i have worked with 3 branches . In the Home-Page I added the home page home page website for both the mobile and desktop versions where the break-point is at 768pxs. I have added the speakers sections dynamically in JavaScript and only to show in the mobile website until you click on the more button to see other speakers and close it by clicking on the less button. I have added a link to the about page and a link in the logo to direct user page to the home page. Also in the menu, at screen size less than 768px the hamburger menu appears and when click the user is able to see the menu and close it using the times sign.. For the about page i have added all the sections for both the mobile and desktop website. I did set up for my project by installing Linters that i would use.

tingamapuro04 committed:
Add README file

tingamapuro04 commented:
[https://www.loom.com/share/6103581683204e31a9667efc8c215123](url)
A recording.

tingamapuro04 committed:
Add a live demo

tingamapuro04 committed:
Merge pull request #3 from tingamapuro04/About-page

About page

BenMukebo commented on README.md:
```diff
@@ -1,72 +1,50 @@
-
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
```
- [ ] Please add descriptive steps from cloning your app to seeing the results displayed on the browser, as this will be of help to non-technical users. You can put the steps to clone and run the project in a new Getting Started section. This includes `git clone ...` , `cd ....` etc

BenMukebo commented on README.md:
```diff
@@ -1,72 +1,50 @@
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
-
```
- [ ] Kindly add the link to the issue page so that other dev can create an issue from a repository 😄🤚🏻

BenMukebo commented on README.md:
```diff
@@ -1,72 +1,50 @@
 ## Acknowledgments
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+- This project design was inspired by a design from Cindy Shin in Behance.
```
- [OPTIONAL ] Great job giving appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/). :partying_face::+1: what do you think to add a link?  🤔 so that the user can have the ability to visit the original design.
-  eg: Original design idea by [Cindy Shin](https://www.behance.net/adagio07) in Behance.

BenMukebo commented on index.html:
```diff
@@ -0,0 +1,137 @@
+        <a href="#">News</a>
+        <a href="#">Campaign</a>
+      </div>
+    </div>
```
- [ ] Great job so far :clap however your HTML code must use semantic tags, which means, 😎   you should place your nav menu option in the `nav` tag and also I recommend you use a list tag  `ul > li`.  it's always a good practice to use `li` to represent an item in a list and a `ul ` to group them,  It could be something like this :point_down: 

    ```HTML 
          <ul>
                <li><a class="about-me" href="about.html">About</a></li>
                <li><a href="#">Program</a></li>
                <li> ....</li>
          </ul>
     ```

- I know you can do it 💪🏻 😄

BenMukebo commented on about.html:
```diff
@@ -0,0 +1,105 @@
+        <a href="#">News</a>
+        <a href="#">Campaign</a>
+      </div>
+    </div>
```
- [ ] Great job so far :clap however your HTML code must use semantic tags, which means, 😎   you should place your nav menu option in the `nav` tag and also I recommend you use a list tag  `ul > li`.  it's always a good practice to use `li` to represent an item in a list and a `ul ` to group them,  It could be something like this :point_down: 

    ```HTML 
          <ul>
                <li><a class="about-me" href="#">About</a></li>
                <li><a href="#">Program</a></li>
                <li> ....</li>
          </ul>
     ```

- I know you can do it 💪🏻 😄

BenMukebo commented on style.css:
```diff
@@ -0,0 +1,529 @@
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
```
- [ ] Kindly make sure to add the missing `border` effect when the user hover any item in the program items section as shown on the image bellow 👇🏻 :

![main](https://user-images.githubusercontent.com/79058364/169443599-e58401a3-358c-4dab-ae4f-b51f3638bbf9.png)

- Your design:

![main31](https://user-images.githubusercontent.com/79058364/170515590-eac5d971-eada-4553-ae2a-714e5e6e90a1.png)

BenMukebo commented on style.css:
```diff
@@ -0,0 +1,529 @@
+  flex-direction: column;
+  align-items: center;
+  gap: 1rem;
+}
```
- [ ] For this section the background is looking okay but you can improve it to look like the design check images below 👇🏻  🤔  I think  you don't need to use the `background-color: var(--first);` instead you can use an image with a `background-repeat: repeat;` property. What do you think?

- [Cindy Shin](https://www.behance.net/adagio07)[Cindy Shin](https://www.behance.net/adagio07) design:

![og](https://user-images.githubusercontent.com/44745855/132837558-b3e87012-6b81-43f7-9b74-dd6a88f8adab.png)

- Your design:

![main31](https://user-images.githubusercontent.com/79058364/170515590-eac5d971-eada-4553-ae2a-714e5e6e90a1.png)

BenMukebo commented on style.css:
```diff
@@ -0,0 +1,529 @@
+  gap: 0.5%;
+  align-items: center;
+
+  /* width: 90%; */
```
- [ ] Please do not leave unused code as a comment in your codebase. this will make your code much noisy, It is always a good practice to keep your code clean from such comments.

BenMukebo commented on style.css:
```diff
@@ -0,0 +1,529 @@
+  font-weight: 700;
+  font-family: 'Poppins', sans-serif;
+
+  /* color: rgba(88,88,88,0.8); */
```
- [ ] Please do not leave unused code as a comment in your codebase. this will make your code much noisy, It is always a good practice to keep your code clean from such comments. 💯 👍🏻

BenMukebo commented on style.css:
```diff
@@ -0,0 +1,529 @@
+  filter: (50px);
+  -webkit-filter: blur(3px); */
+
+  /* background: #cbc7c753; */
```
- [ ] Please do not leave unused code as a comment in your codebase. this will make your code much noisy, It is always a good practice to keep your code clean from such comments. 💯 👍🏻

BenMukebo commented on style.css:
```diff
@@ -0,0 +1,529 @@
+  display: flex;
+  justify-content: space-between;
+
+  /* background: #cbc7c753; */
```
- [ ] Please do not leave unused code as a comment in your codebase. this will make your code much noisy, It is always a good practice to keep your code clean from such comments. 💯 👍🏻

BenMukebo commented on index.html:
```diff
@@ -0,0 +1,137 @@
+          <img class="ignite-img" src="./imgs/ignite.png" alt="ignite icon">
+          <h2>ignite</h2>
+          <p>Get oppurtunities to network with different people from all over the world.</p>
+        </div>
```
- [ ]  The HTML uses semantic tags, sections use `section`, articles use `article`..., Kindly for each program article place then in article tags and not divs. `<article class="item"></article>` 👍🏻 It could be something like this :point_down:

```
   <article class="workshop">
          <img class="workshop-img" src="./imgs/workshop1.jfif" alt="workshop icon">
          <h2>Workshop</h2>
          <p>Try creating your own work using open source license rather than just watching</p>
   </article>
```



- I know you can do it 😄 💪🏻

BenMukebo commented on style.css:
```diff
@@ -0,0 +1,529 @@
+}
+
+@media all and (min-width: 768px) {
+  #pages {
+    display: flex;
+    flex-direction: row;
+    background: whitesmoke;
+    height: auto;
+    align-items: center;
+    justify-content: space-around;
+    font-size: 1.3rem;
+    font-weight: 400;
+  }
+
```
- [ ] Kindly add more padding or margin-left and padding or margin-right  to your `header > div#pages` :sunglasses::nerd_face:
- [ ] You can consider applying this to the About Page since they have the same header  🎨 🖌️  I know you can do it :muscle: 😄 

- Your Design:

![nav32](https://user-images.githubusercontent.com/79058364/170520884-7e448d2e-d1d4-42fb-b3ff-ad0859e1c31d.png)



- [Cindy Shin](https://www.behance.net/adagio07)[Cindy Shin](https://www.behance.net/adagio07) design:

![nav31](https://user-images.githubusercontent.com/79058364/170520928-af139575-b4c9-4cbd-9b64-5849203be28f.png)

BenMukebo commented on index.html:
```diff
@@ -0,0 +1,137 @@
+      <a href="index.html"><img class="uon-logo-f" src="./imgs/uonlogo.png" alt="University of nairobi Logo"></a>
+      
+      <div class="pages-a">
+        <a class="about-me" href="about.html">About</a>
+        <a href="#">Program</a>
+        <a href="#">Join</a>
+        <a href="#">Sponsor</a>
+        <a href="#">News</a>
+        <a href="#">Campaign</a>
```
- [ ] According to the project requirement, The pages should look almost identical to the original design, outside the content differences. what if you add this red border to the last menu option of your navbar and don't forget to use the same border-color and border-width as from the original template 💯 😄 
- [ ] Please change his text color too 👇🏻  

- [Cindy Shin](https://www.behance.net/adagio07)[Cindy Shin](https://www.behance.net/adagio07) design:

![nav1](https://user-images.githubusercontent.com/79058364/168643439-051c8a69-818b-47fe-af98-bde68f57e2dc.png)

- Your design:

![navb32](https://user-images.githubusercontent.com/79058364/170523267-8bf7d813-226b-46a5-a90f-05ef5a5b5dcc.png)

BenMukebo replied:
- [ ] Kindly make sure  When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed. 😄 
- [ ] Kindly make sure When the user clicks (or taps) any option of the mobile menu, the mobile menu disappears. 😄 

- [ ] What about preventing the sub-background from scrolling when the navigation bar is open? 🤔  You can use an `over-flow: hidden;` CSS property to do this.

BenMukebo commented on index.html:
```diff
@@ -0,0 +1,137 @@
+  </header>
+
+  <main>
+    <section class="first">
+      <div class="intro">
+        <h3>"Hello! It is Here Again"</h3>
+        <p class="title">NAIROBI INNOVATION WEEK EDITION 3 2022</p>
+        <p class="short-info">An Interactive session about innovation in Kenya and Africa brought to you by the University of Naorobi. This is to promote advanced changes in our continent.</p>
+        <p class="date">2022.04.26(WED) - 28(FRI)</p>
+        <p class="location">@ University of Nairobi, Great Court Main Campus.</p>
+      </div>
+      <div class="overle"></div>
+    </section>
```
- [ ] Please change the `background image` of your first section as per requirements you should follow the design from [behance](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) which means colors, font-family, background etc.. but you were free to personalize just contents. kindly do that even on the About Page. innocent 🙏🏻




- [Cindy Shin](https://www.behance.net/adagio07)[Cindy Shin](https://www.behance.net/adagio07) design:

![bs](https://user-images.githubusercontent.com/44745855/132836775-ef27cc32-7245-474d-9dc3-cea8a091e864.png)

- Your design:

![main32](https://user-images.githubusercontent.com/79058364/170523858-8469bc6d-488d-441f-94d9-857537192658.png)

BenMukebo commented on about.html:
```diff
@@ -0,0 +1,105 @@
+        <a href="https://www.youtube.com/watch?v=-M4SNHmqWG8"><img src="./imgs/photo18.jfif" alt=""></a>
+        <a href="https://www.youtube.com/watch?v=-yToP3YiR0I"><img src="./imgs/photo20.jfif" alt=""></a>
+      </div>
+    </section>
```
- [ ] Also to improve your UI design, you could add the background overlay color on your maps 👍 

- Your Design:

![about32](https://user-images.githubusercontent.com/79058364/170525696-af2c7bce-e6cf-4c55-b7f8-b439c37be917.png)


- [Cindy Shin](https://www.behance.net/adagio07)[Cindy Shin](https://www.behance.net/adagio07) design:

![Annotation 2022-05-18 110524](https://user-images.githubusercontent.com/85113567/169014609-2f09ccf3-cab1-4dda-b381-8e449c302275.jpg)

BenMukebo commented on about.html:
```diff
@@ -0,0 +1,105 @@
+
+  <footer class= "A-footer">
+    <div id="theme">
+      <!-- <p class="theme">NAIROBI INNOVATION WEEK</p> -->
```
- [ ] Please do not leave unused code as a comment in your codebase. this will make your code much noisy, It is always a good practice to keep your code clean from such comments.

BenMukebo commented on app.js:
```diff
@@ -0,0 +1,118 @@
+  },
+];
+
+const speakers1 = [
+  {
+    check: './imgs/check.png',
+    image: './imgs/njoroge.jpg',
+    fullName: 'Patrick Njoroge',
+    position: 'Central Bank Of Kenya governor, Economist',
+    description: 'Patrick Ngugi Njoroge is a Kenyan economist, banker and the ninth governor of the Central Bank of Kenya.',
+  },
+  {
+    check: './imgs/check.png',
+    image: './imgs/koome.jpg',
+    fullName: 'Martha Koome',
+    position: 'Kenyan Chief Justice',
+    description: 'Martha Karambu Koome is a Kenyan advocate, currently serving as the Chief Justice of Kenya, and is the first woman to occupy the post.',
+  },
```
- [ ] Great job generating the "Featured speakers" dynamically with JavaScript. The section. :partying_face::+1:💪🏻 

BenMukebo commented on style.css:
```diff
@@ -0,0 +1,529 @@
+@import url('http://fonts.cdnfonts.com/css/cocogoose');
+
+:root {
+  --first: #272a31;
+  --second: #ec5242;
+  --third: #d3d3d3;
+}
```
- [ ] Excellent work implementing CSS variables for reducing repetition in CSS, and also for powerful runtime effects :partying_face::+1: 💪🏻

BenMukebo commented on about.css:
```diff
@@ -0,0 +1,345 @@
+@import url('http://fonts.cdnfonts.com/css/cocogoose');
+
+:root {
+  --first: #272a31;
+  --second: #ec5242;
+  --third: #d3d3d3;
+}
+
```
- [ ] Excellent work implementing CSS variables for reducing repetition in CSS, and also for powerful runtime effects :partying_face::+1: 💪🏻

BenMukebo commented on index.html:
```diff
@@ -0,0 +1,137 @@
+      <hr>
+      <div id="wageni"></div>
+      <div id="wageni1"></div>
+    </section>
```
- [ ] Great job generating the "Featured speakers" dynamically with JavaScript. The section. :partying_face::+1:💪🏻 

BenMukebo reviewed (changes requested):
### Hi @tingamapuro04,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Your repo looks Professional  👍🏻 
- Your design looks fine :rocket: 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

- [ ] Great job recording a video presenting the features of the project 👍🏻 However, the link is broken and remember According to the project requirement, 
- 1 The video presentation should contain a highlight about a piece of the code you finds interesting or is proud of making. 
- 2 The maximum length of the video is 5 minutes.
- 3 The video contains a clear and objective description about the project in professional English. The topic and purpose of the project are made clear.
- 4 The video contains a demo of the project, showing the pages in different screen sizes.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ OPTIONAL] Please consider adding the link to the video presenting the features of the project. into your README file. that will make increase the description and will your README file more professional 😄 💯 

Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

BenMukebo commented on about.html:
```diff
@@ -0,0 +1,105 @@
+      </div>
+    </section>
+
+    <section class="A-four">
+      <h2>Partner</h2>
+      <hr>
+      <div class="A-sponsors">
+        <img src="./imgs/kcb.png" alt="KCB bank">
+        <img src="./imgs/safaricom.png" alt="Safaricom">
+        <img src="./imgs/microsoft.png" alt="microsoft">
+        <img src="./imgs/udemy.png" alt="Udemy">
+      </div>
+    </section>
```
- [ OPTIONAL] Great job so far 👏🏻 , however, you could improve this section by using semantic  HTML code what do you think to use a list tag  `ul > li`.  it's always a good practice to use `li` to represent an item in a list and a `ul ` to group them,  It could be something like this :point_down: 
    ```HTML 
          <ul>
                <li><a class="about-me" href="#">About</a></li>
                <li><a href="#">Program</a></li>
                <li> ....</li>
          </ul>
     ```

- I know you can do it 💪🏻 😄

BenMukebo commented on about.html:
```diff
@@ -0,0 +1,105 @@
+  </main>
+
+
+  <footer class= "A-footer">
+    <div id="theme">
+      <!-- <p class="theme">NAIROBI INNOVATION WEEK</p> -->
+      <img src="./imgs/niw2019.jfif" alt="">
+    </div>
+    <div class="last_info">
+      <h4>2022 Nairobi Innovation Week, Some Rights Reserved</h4>
+      <p class="footer-para">The annual Nairobi Innovation Week (NIW) was launched in 2015 to promote Kenya’s and the world’s innovation and entrepreneurial ecosystems. Throughout the years, The University</p>
+    </div>
+
+  </footer>
```
- [ ] Great job so far but why do you use different logos on the About and Home page? I think your footer should be the same and should have the same logo on those two pages since it's the same webpage, kindly change fix. :thanks :innocent::upside_down_face:

- Home Page:

![fothom](https://user-images.githubusercontent.com/79058364/170529671-27d18f89-ef17-420e-b54e-d27fa8142a67.png)


- About Page:

![fotab](https://user-images.githubusercontent.com/79058364/170529861-da512ebc-4709-4d30-97c0-d22204c415a9.png)

tingamapuro04 committed:
various minor changes

tingamapuro04 committed:
Correction suggested by reviewer on the nav bar

tingamapuro04 committed:
Linter corrections

tingamapuro04 committed:
Add margin to nav bar

tingamapuro04 committed:
Add background images

tingamapuro04 committed:
Add padding to UoNSchool

tingamapuro04 committed:
Add type to buttons

tingamapuro04 committed:
Add type to buttons

tingamapuro04 committed:
Add noopener attribute

tingamapuro04 committed:
Linter corrections

tingamapuro04 commented:
I have done changes on Home-Page branch some of which include;
1  In the header i have used sematic tags nav instead of div and used ul instead of just listing items singularly.
2 In the programs section I have added a dark background image, hover effect where the border becomes white and each program has been put inside an article tag
3 in the events section i have added a color overlay  with linear gradient
4 I have made the logo in the footer section the same 
5 In the readme file i have added more descriptions such as, I have included a link to the issue and steps to follow to get the repo running in the user machine.

A link to the 5 minutes description video  ==> [https://www.loom.com/share/f44065910d054fefb3f1ed0e4260e6bb](url)

Ismailco commented on README.md:
```diff
@@ -1,72 +1,61 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
```
- [ ] [Optional] Please consider adding all your comments that describe a change to the description of the Pull Request (first comment)

tingamapuro04 replied:
I don't understand what is requested of me, please may you help me with it. Thanks.
.

Ismailco commented on README.md:
```diff
@@ -1,72 +1,61 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
```
- [ ] Kindly fix the video URL, also the video link should be on the README for better documentation, Please remove the first loom video link and add the new one to the README

Ismailco commented on style.css:
```diff
@@ -0,0 +1,680 @@
+  background: var(--second);
+}
+
+.programs article {
```
- [ ] good work 👍🏼 implementing the border as the first reviewer suggest however the middle program items broke when hovering over them, please consider fixing them

![Screenshot_20220528_045943](https://user-images.githubusercontent.com/25227022/170809102-e929b094-1874-4420-8920-7cab19ab6294.png)

Ismailco commented on index.html:
```diff
@@ -0,0 +1,142 @@
+    <section class="four" id="sponsor">
+      <h2>partner</h2>
+      <hr class="sponsor-hr">
+      <div class="sponsors">
```
- [ ] [Optional] This is optional but you can use semantic tags instead of ```<div>``` when you can, this will improve the code readability and SEO

Ismailco commented on about.html:
```diff
@@ -0,0 +1,106 @@
+      </div>
+      <a href="index.html" class="brand"><img src="./imgs/uonlogo.png" alt="uon logo"></a>
+      <ul class="navmenu" id="navmenu">
+          <li class="items item1"><a href="about.html" class="about-link" id="about">About</a></li>
```
- [ ] [OPTIONAL] You don't have a link to the home page from the about page beside the logo which is not very obvious to use, please consider adding a link to the home page. you can use the about link for that e.g. 
   ```
     <li class="items item1">
          <a href="index.html" class="about-link" id="about">Home</a>
      </li>
  ```

Ismailco commented on about.html:
```diff
@@ -0,0 +1,106 @@
+    <section class="A-four">
+      <h2>Partner</h2>
+      <hr>
+      <div class="A-sponsors">
+        <img src="./imgs/kcb.png" alt="KCB bank">
+        <img src="./imgs/safaricom.png" alt="Safaricom">
+        <img src="./imgs/microsoft.png" alt="microsoft">
+        <img src="./imgs/udemy.png" alt="Udemy">
+      </div>
```
- [ ] [OPTIONAL] You can choose a darker logo for the partner section so your design is as close to the CC design as possible, don't forget to implement this on the home page also

|Your design|CC design|
|-|-|
|![Screenshot_20220528_052128](https://user-images.githubusercontent.com/25227022/170809632-6ccce78b-cca8-49a8-8d22-90b98c2e5ad6.png)|![Screenshot_20220528_052518](https://user-images.githubusercontent.com/25227022/170809696-859b1bb1-b0a9-45aa-a905-d0357546612e.png)|

Ismailco commented on index.html:
```diff
@@ -0,0 +1,142 @@
+
+    <section class="second" id="programs">
+      <h2>Main Program</h2>
+      <hr>
```
- [ ] [OPTIONAL] The color of the dividers is not the similar to the original design, please stick to the design as much as possible

|Your design|CC design|
|-|-|
|![Screenshot_20220528_052413](https://user-images.githubusercontent.com/25227022/170809915-25bc42e0-b4a3-4808-b6e0-08a49b92c2e2.png)|![Screenshot_20220528_052450](https://user-images.githubusercontent.com/25227022/170810024-cab463cf-140e-4da5-b7f6-3b392b0e795e.png)|
|![Screenshot_20220528_052424](https://user-images.githubusercontent.com/25227022/170809988-d99af6c8-dca4-4e7a-9041-4c0f4631d4f5.png)|![Screenshot_20220528_052504](https://user-images.githubusercontent.com/25227022/170809995-ec5dd711-9183-4ad5-9c60-1a317c7bb494.png)|

Ismailco commented on index.html:
```diff
@@ -0,0 +1,142 @@
+  </header>
+
+  <main>
+    <section class="first">
```
- [ ] [OPTIONAL] This is a suggestion based on your video, you can use any image just add an overlay with the same color of the original design, and some opacity. you can do the same for your about page

Ismailco reviewed (changes requested):
#### Hi @tingamapuro04,

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![giphy](https://user-images.githubusercontent.com/25227022/169399289-510de072-42e7-419b-8c2c-da9344bdd430.gif)

## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

tingamapuro04 committed:
Repair programs section

tingamapuro04 committed:
Linter corrections

tingamapuro04 committed:
Add hr styling

tingamapuro04 commented:
I have worked on the Home-Page branch I have fixed the program section such that when you hover over it it does not break and I also added a background image. In the sponsors' section, I have added a darker background color. In the readme file i have added the link to the five-minute video to the live demo link. I am not very sure of the problem with the link to the video because from my side it is working perfectly fine and if there is any issue with this one let me know how I need to fix it if you don't mind. I am reposting it here [https://www.loom.com/share/f44065910d054fefb3f1ed0e4260e6bb](url)

Ismailco commented:
@tingamapuro04, Please implement all the changes, I can't approve your project, but I can wait for you to implement all the changes, and please add the website link to the README, not just the video. this is your last review that's why I contacted you first

Ismailco commented on about.html:
```diff
@@ -0,0 +1,106 @@
+      </div>
+    </section>
+
+    <section class="A-four">
+      <h2>Partner</h2>
+      <hr>
+      <div class="A-sponsors">
+        <img src="./imgs/kcb.png" alt="KCB bank">
+        <img src="./imgs/safaricom.png" alt="Safaricom">
+        <img src="./imgs/microsoft.png" alt="microsoft">
+        <img src="./imgs/udemy.png" alt="Udemy">
+      </div>
```
- [ ] [OPTIONAL] You can choose a darker logo for the partner section so your design is as close to the CC design as possible, don't forget to implement this on the home page also.

|Your design|CC design|
|-|-|
|![Screenshot_20220528_052128](https://user-images.githubusercontent.com/25227022/170809632-6ccce78b-cca8-49a8-8d22-90b98c2e5ad6.png)|![Screenshot_20220528_052518](https://user-images.githubusercontent.com/25227022/170809696-859b1bb1-b0a9-45aa-a905-d0357546612e.png)|

tingamapuro04 committed:
Add link to home page

tingamapuro04 commented:
On the Home-Page branch, on the about page i have added a link to the home page so that users can navigate back to the Home page. I have also added a link to the project on the readme file. @Ismailco I can see most request are optional and so i don't know which one i must work on. For the first request you made (Please consider adding all your comments that describe a change to the description of the Pull Request (first comment)) I don't understand what it is maybe we can have a moment on zoom so that you could show me how to go about it

Ismailco reviewed (approved):
### Hi @tingamapuro04,

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

![cr-approve](https://user-images.githubusercontent.com/25227022/165483662-7b06c812-4cc0-40b9-a84c-4729d4cf10c3.gif)

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

tingamapuro04 commented:
Thank you for the feedback. I like connecting with my code reviewers would
you like to ?

On Sat, 28 May 2022, 22:01 ismail courr, ***@***.***> wrote:

> ***@***.**** approved this pull request.
> Hi @tingamapuro04 <https://github.com/tingamapuro04>,
>
> In my opinion, your project is ready for the presentation! There is
> nothing else to say other than... it's time to merge it [image: :shipit:]
>
> [image: cr-approve]
> <https://user-images.githubusercontent.com/25227022/165483662-7b06c812-4cc0-40b9-a84c-4729d4cf10c3.gif>
>
> Good luck! 🎉
>
> Feel free to leave any questions or comments in the PR thread if something
> is not 100% clear.
> ------------------------------
>
> In about.html
> <https://github.com/tingamapuro04/Nairobi-Innovation-Week/pull/1#discussion_r884166420>
> :
>
> > +    <section class="A-four">
>
> +      <h2>Partner</h2>
>
> +      <hr>
>
> +      <div class="A-sponsors">
>
> +        <img src="./imgs/kcb.png" alt="KCB bank">
>
> +        <img src="./imgs/safaricom.png" alt="Safaricom">
>
> +        <img src="./imgs/microsoft.png" alt="microsoft">
>
> +        <img src="./imgs/udemy.png" alt="Udemy">
>
> +      </div>
>
>
>
>    - [OPTIONAL] You can choose a darker logo for the partner section so
>    your design is as close to the CC design as possible, don't forget to
>    implement this on the home page also.
>
> Your design CC design
> [image: Screenshot_20220528_052128]
> <https://user-images.githubusercontent.com/25227022/170809632-6ccce78b-cca8-49a8-8d22-90b98c2e5ad6.png> [image:
> Screenshot_20220528_052518]
> <https://user-images.githubusercontent.com/25227022/170809696-859b1bb1-b0a9-45aa-a905-d0357546612e.png>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/tingamapuro04/Nairobi-Innovation-Week/pull/1#pullrequestreview-988547216>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ATZCTJ4EHORISFJQSGTGDHDVMJUPDANCNFSM5WV2O5UA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

Ismailco commented:
> Thank you for the feedback. I like connecting with my code reviewers would
> you like to ?
> 
> On Sat, 28 May 2022, 22:01 ismail courr, ***@***.***> wrote:
> 
> > ***@***.**** approved this pull request.
> > Hi @tingamapuro04 <https://github.com/tingamapuro04>,
> >
> > In my opinion, your project is ready for the presentation! There is
> > nothing else to say other than... it's time to merge it [image: :shipit:]
> >
> > [image: cr-approve]
> > <https://user-images.githubusercontent.com/25227022/165483662-7b06c812-4cc0-40b9-a84c-4729d4cf10c3.gif>
> >
> > Good luck! 🎉
> >
> > Feel free to leave any questions or comments in the PR thread if something
> > is not 100% clear.
> > ------------------------------
> >
> > In about.html
> > <https://github.com/tingamapuro04/Nairobi-Innovation-Week/pull/1#discussion_r884166420>
> > :
> >
> > > +    <section class="A-four">
> >
> > +      <h2>Partner</h2>
> >
> > +      <hr>
> >
> > +      <div class="A-sponsors">
> >
> > +        <img src="./imgs/kcb.png" alt="KCB bank">
> >
> > +        <img src="./imgs/safaricom.png" alt="Safaricom">
> >
> > +        <img src="./imgs/microsoft.png" alt="microsoft">
> >
> > +        <img src="./imgs/udemy.png" alt="Udemy">
> >
> > +      </div>
> >
> >
> >
> >    - [OPTIONAL] You can choose a darker logo for the partner section so
> >    your design is as close to the CC design as possible, don't forget to
> >    implement this on the home page also.
> >
> > Your design CC design
> > [image: Screenshot_20220528_052128]
> > <https://user-images.githubusercontent.com/25227022/170809632-6ccce78b-cca8-49a8-8d22-90b98c2e5ad6.png> [image:
> > Screenshot_20220528_052518]
> > <https://user-images.githubusercontent.com/25227022/170809696-859b1bb1-b0a9-45aa-a905-d0357546612e.png>
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <https://github.com/tingamapuro04/Nairobi-Innovation-Week/pull/1#pullrequestreview-988547216>,
> > or unsubscribe
> > <https://github.com/notifications/unsubscribe-auth/ATZCTJ4EHORISFJQSGTGDHDVMJUPDANCNFSM5WV2O5UA>
> > .
> > You are receiving this because you were mentioned. Message ID:
> > ***@***.***>
> >
> 

if you need quick help, am happy to assist 😊

Ismailco commented:
> Thank you for the feedback. I like connecting with my code reviewers would
> you like to ?
> 

If you need a quick assist, am happy to help 😊

tingamapuro04 commented:
Hello sir can we have a moment for a short zoom call?


On Sat, May 28, 2022 at 9:20 PM ismail courr ***@***.***>
wrote:

> @tingamapuro04 <https://github.com/tingamapuro04>, Please implement all
> the changes, I can't approve your project, but I can wait for you to
> implement all the changes, and please add the video link and website link
> to the README. this is your last review that's why I contacted you first
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/tingamapuro04/Nairobi-Innovation-Week/pull/1#issuecomment-1140309005>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ATZCTJ2P52FRZ34G3N7CNPTVMJPW7ANCNFSM5WV2O5UA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
